contributed_definitions/nyaml/NXpeak.yaml

```
@@ -1,59 +1,140 @@
 category: base
 doc: |
-  Description of peaks, their functional form or measured support.
+  Base class for describing spectral peaks, their functional form, and support values
+  (i.e., the discretization (points) at which the function has been evaluated).
```

I would just describe one peak, and define one instance per peak.
```suggestion
  Base class for describing a spectral peak, its functional form, and support values
  (i.e., the discretization (points) at which the function has been evaluated).
```

---

Thanks, sounds good.